birdhouse/components/README.rst

```diff
@@ -12,6 +12,9 @@ The scheduler component runs specific jobs on a schedule. This is similar to usi
 service but this runs in docker containers and is specifically designed to interact with the
 Birdhouse stack.
 
+Available jobs
+-------------
```

Comment:
The file is rendering properly on Github now.  So it's the addition of this 2nd level that fixes the rendering?

Reply:
Yeah. RST doesn't use a specific character to infer heading levels, but rather, relies on the order by which they are specified in the file.